bot/seasons/season.py

```diff
@@ -161,20 +312,176 @@ def __init__(self, bot):
             self.sleep_time = 86400  # next time, sleep for 24 hours.
 
             # If the season has changed, load it.
-            new_season = get_season(self.bot, date=datetime.date.today())
+            new_season = get_season(date=datetime.datetime.utcnow())
             if new_season != self.season:
                 await self.season.load()
 
     @with_role(Roles.moderator, Roles.admin, Roles.owner)
-    @commands.command(name='season')
+    @commands.command(name="season")
     async def change_season(self, ctx, new_season: str):
         """
         Changes the currently active season on the bot.
         """
 
-        self.season = get_season(self.bot, season_name=new_season)
+        self.season = get_season(season_name=new_season)
         await self.season.load()
         await ctx.send(f"Season changed to {new_season}.")
 
+    @with_role(Roles.moderator, Roles.admin, Roles.owner)
+    @commands.command(name="seasons")
+    async def show_seasons(self, ctx):
+        """
+        Shows the available seasons and their dates.
+        """
+
+        # sort by start order, followed by lower duration
+        def season_key(season: SeasonBase):
+            return season.start(), season.end() - datetime.datetime.max
+
+        current_season = self.season.name
+
+        forced_space = "\u200b "
+
+        entries = []
+        seasons = [get_season_class(s) for s in get_seasons()]
+        for season in sorted(seasons, key=season_key):
+            start = season.start_date
+            end = season.end_date
+            if start and not end:
+                period = f"From {start}"
+            elif end and not start:
+                period = f"Until {end}"
+            elif not end and not start:
+                period = f"Always"
+            else:
+                period = f"{start} to {end}"
+
+            # bold period if current date matches season date range
+            is_current = season.is_between_dates(datetime.datetime.utcnow())
+            pdec = "**" if is_current else ""
+
+            # underline currently active season
+            is_active = current_season == season.name
+            sdec = "__" if is_active else ""
+
+            entries.append(
+                f"**{sdec}{season.__name__}:{sdec}**\n"
+                f"{forced_space*3}{pdec}{period}{pdec}"
```

Comment:
```suggestion
                f"{forced_space*3}{pdec}{period}{pdec}\n"
```

I think this would look better with an empty line between the seasons.

Reply:
Sure thing. Done:
![image](https://user-images.githubusercontent.com/29337040/49344340-8da01600-f6c1-11e8-860c-1b05f4ab8d60.png)

Reply:
looks great!